lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_build_preflight.rb

```diff
@@ -33,7 +33,6 @@ def self.run(params)
           raise
         end
 
-        Action.sh('rake dependencies:pod:clean')
```

Comment:
Also, I feel good about removing the implicit assumption that a client would have a `Rakefile` with this task defined in it.

Luckily, this was the only direct call to `rake` I could find. There is another one, above, but is in a comment.

https://github.com/wordpress-mobile/release-toolkit/blob/9f78ca93a1d4c494db90e52abbf090126355f9b1/lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_build_preflight.rb#L32